relnotes/pkg-iteration.feature.md

```diff
@@ -0,0 +1,21 @@
+## The package iteration can now be specified via `PKGITERATION`
+
+Package versions consist of two components: the _upstream version_, and
+the _package iteration_ (corresponding to the `debian_revision` where
+deb packages are concerned).
+
+Previously the package iteration was hardcoded to match the distro code
+(e.g. `xenial`), disallowing typical patterns referencing the packager
+and changes to the packaging rather than the upstream project.  For
+example, Ubuntu packages typically use an `XubuntuY` format where `X`
+is the numerical revision of the Debian package that has been forked
+(or `0` if there is no such package), and `Y` is the Ubuntu package
+revision (hence Ubuntu package versions like `2.7.0-2ubuntu3.1`, where
+`2.7.0` is the upstream version and `2ubuntu3.1` means Ubuntu revision
+3.1 of Debian package revision 2).
+
+The new `PKGITERATION` variable allows the package iteration to be set
+to whatever is wanted for the package, whether via `Config.mak` or the
+command line.  Note that if you want to still include the distro code,
+this will have to be included manually as part of the custom iteration
+choice, for example: `PKGITERATION=0tsunami0~"$(lsb_release -cs)"`.
```

Comment:
I would also use a more simple example, like `PKGITERATION=1` because unless you have the whole explanation about what each number represents there, it's just cryptic. And if you follow debian packaging guidelines, the examples will just have `1` as the debian revision. I have the feeling the whole thing about Ubuntu is just guiding users to an unnecessarily complex scheme.